invideoquiz/public/js/src/invideoquiz.js

```diff
@@ -98,6 +125,19 @@ function InVideoQuizXBlock(runtime, element) {
         video.on('pause', function () {
           clearInterval(intervalObject);
           if (problemToDisplay) {
+            resizeIntervalObject = setInterval(function () {
+
+              // check if the size has changed from the previous state; if so, update
+              // both our known size measurements and the size of the problem
+              currentVideoDimensions = getVideoDimensions();
+
+              if ((currentVideoDimensions[0] !== knownVideoDimensions[0]) ||
```

Comment:
this conditional can now be:

`if (videoDimensionsDiffer(currentVideoDimensions, knownVideoDimensions)) {`